configs/swarm/genesis.json

```diff
@@ -121,48 +121,6 @@
         }
       }
     },
-    {
```

Comment:
Why there is no parameters in genesis? Shouldn't we enforce user to provide on-chain parameters?

Reply:
should we force this? I was thinking that using defaults would be more simple

Reply:
opened #4780